experimental/dds/tree2/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -20,15 +20,23 @@ import {
 } from "../../../core";
 import { brand } from "../../../util";
 import { SharedTreeTestFactory, toJsonableTree, validateTree } from "../../utils";
+import { SharedTreeView } from "../../../shared-tree";
 import { makeOpGenerator, EditGeneratorOpWeights, FuzzTestState } from "./fuzzEditGenerators";
-import { fuzzReducer } from "./fuzzEditReducers";
+import { fuzzComposedVsIndividualReducer, fuzzReducer } from "./fuzzEditReducers";
 import { onCreate, initialTreeState } from "./fuzzUtils";
 import { Operation, TreeOperation } from "./operationTypes";
 
 interface AbortFuzzTestState extends FuzzTestState {
 	firstAnchor?: Anchor;
 }
 
+/**
+ * This interface is meant to be used for tests that require you to store a branch of a tree
+ */
+export interface BranchedTreeFuzzTestState extends FuzzTestState {
+	branch?: SharedTreeView;
+}
+
```

Comment:
I named this interface `BranchedTreeFuzzTestState` as I wasn't sure if there would be other test scenarios that would need a branch in it's state. If not, I could rename this to something more specific like `ComposeVsIndividualFuzzTestState`